addons/mail/models/mail_notification.py

```diff
@@ -111,12 +111,17 @@ def format_failure_reason(self):
     # DISCUSS
     # ------------------------------------------------------------
 
+    def _is_display_notification(self):
```

Comment:
I feel like it's a bit overkill to add a field just for this particular use case

Many time in mail we hardcode `mt_comment` or `mt_note` so maybe we can remove the field and check manually the id of `subtype_id` ?

What do you think ?

Reply:
Following our discussion, we will keep the added field display_notification in mail.message.subtype so that it can be configured by the user (use-case validated).